test/server/overload_manager_impl_test.cc

```diff
@@ -545,19 +556,32 @@ TEST_F(OverloadManagerImplTest, CreateScaledTimerWithAbsoluteMinimum) {
 
   auto* mock_scaled_timer = new Event::MockTimer();
   MockFunction<Event::TimerCb> mock_callback;
-  EXPECT_CALL(*scaled_timer_manager, createTimer_)
-      .WillOnce([&](Event::ScaledTimerMinimum minimum, auto) {
-        // This timer was created with an absolute minimum. Test that by checking an arbitrary
-        // value.
-        EXPECT_EQ(minimum.computeMinimum(std::chrono::seconds(55)), std::chrono::seconds(2));
-        return mock_scaled_timer;
-      });
+  EXPECT_CALL(*scaled_timer_manager,
+              createTimer_(Eq(Event::AbsoluteMinimum(std::chrono::seconds(2))), _))
+      .WillOnce(Return(mock_scaled_timer));
```

Comment:
The previous 2 test cases seem redundant with OverloadManagerImplTest.TimerTypesProduceCorrectMinimums below.

Reply:
Agreed. Removed.